workflows/cellranger-quant/run-cellranger.nf

```diff
@@ -17,22 +16,20 @@ cellranger_tech_list = ["10Xv2", "10Xv3", "10Xv3.1", "10Xv2_5prime"]
 spatial_techs = ["spatial", "visium_v1", "visium_v2"]
 all_tech_list = cellranger_tech_list + spatial_techs
 
-// build full paths
-params.index_path = "${params.ref_dir}/${params.index_dir}/${params.index_name}"
-
 process cellranger{
   container '589864003899.dkr.ecr.us-east-1.amazonaws.com/scpca-cellranger:6.1.2'
   publishDir "${params.outdir}", mode: 'copy'
-  tag "${meta.scpca_run_id}-${index}" // add tag for tracking sample names in trace file  
+  tag "${meta.scpca_run_id}-${index_name}" // add tag for tracking sample names in trace file  
   label 'cpus_8'
   label 'bigdisk'
   input:
     tuple val(meta), path(fastq_dir), val(include_introns)
+    val index_name
     path index
```

Comment:
I'd probably make this a tuple for organizational logic purposes, though I don't think it matters much.
```suggestion
    tuple val(index_name), path(index)
```